### PR TITLE
Tim's card initial commit

### PR DESCRIPTION
**Pull request by tcas2162**
Added my card information. Included links to three Youtube channels that have helped me on my coding journey

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8105,37 +8282,88 @@ <h2>A project for first-time contributions</h2>
...
+                <a href="#" target="_blank" title="Third resource">
+                  Resource 3
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*CZ's card END ________  -->
+
+        <!-- ________ *Tim's card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:8317]: It looks like you've added your card at the end of the page.

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Your commit has re-formatted the previous code, by adding whitespaces and new lines.

**Comment by Preet-Sojitra** [comment]
@tcas2162 your pr is causing around 40 merge conflicts, which is kinda difficult to solve. So you are requested to open a new pull request. Once you open a new pull request, we will merge it. 
Thank you

**Comment by win21H2** [comment]
Will close this PR and try to injest the HTML code. REASON FOR CLOSE: This issue has been open and changes have not been made since March 24

**Pull request closed without merging**